crowbar_framework/config/routes.rb

```diff
@@ -24,102 +24,103 @@
+  get "crowbar/:controller/1.0(.:format)", action: "index", as: "index_barclamp"
+  delete "crowbar/:controller/1.0/:id(.:format)", action: "delete", constraints: { id: /[^\/]+/ }, as: "delete_barclamp"
+  get "crowbar/:controller/1.0/:id(.:format)", action: "show", constraints: { id: /[^\/]+/ }, as: "show_barclamp"
+  get "crowbar/:controller(.:format)", action: "versions", as: "versions_barclamp"
+  post "crowbar/:controller/1.0/:action/:id(.:format)", constraints: { id: /[^\/]+/ }, as: "action_barclamp"
+  get "crowbar(.:format)", controller: "barclamp", action: "barclamp_index", as: "barclamp_index_barclamp"
+  get "crowbar/modules/1.0(.:format)", controller: "barclamp", action: "modules", as: "barclamp_modules"
 
-  get "crowbar/:barclamp/1.0/help", action: "help", controller: "barclamp"
-  get "crowbar/:barclamp/1.0/proposals/nodes", controller: "barclamp", action: "nodes"
-  put "crowbar/:barclamp/1.0/proposals", action: "proposal_create", controller: "barclamp"
-  get "crowbar/:barclamp/1.0/proposals", action: "proposals", controller: "barclamp"
-  post "crowbar/:barclamp/1.0/proposals/commit/:id", action: "proposal_commit", controller: "barclamp"
+  get "crowbar/:barclamp/1.0/help(.:format)", action: "help", controller: "barclamp"
+  get "crowbar/:barclamp/1.0/proposals/nodes(.:format)", controller: "barclamp", action: "nodes"
+  put "crowbar/:barclamp/1.0/proposals(.:format)", action: "proposal_create", controller: "barclamp"
+  get "crowbar/:barclamp/1.0/proposals(.:format)", action: "proposals", controller: "barclamp"
+  post "crowbar/:barclamp/1.0/proposals/commit/:id(.:format)", action: "proposal_commit", controller: "barclamp"
   get "crowbar/:barclamp/1.0/proposals/status(.:format)", action: "proposal_status", controller: "barclamp"
-  delete "crowbar/:barclamp/1.0/proposals/:id", action: "proposal_delete", controller: "barclamp"
-  post "crowbar/:barclamp/1.0/proposals/:id", action: "proposal_update", controller: "barclamp"
-  get "crowbar/:barclamp/1.0/proposals/:id", action: "proposal_show", controller: "barclamp"
-  get "crowbar/:barclamp/1.0/elements", action: "elements", controller: "barclamp"
-  get "crowbar/:barclamp/1.0/elements/:id", action: "element_info", controller: "barclamp"
-  post "crowbar/:barclamp/1.0/transition/:id", action: "transition", controller: "barclamp"
-  get "crowbar/:barclamp/1.0/transition/:id", action: "transition", controller: "barclamp"
-  get "crowbar/:barclamp/1.0", action: "index", controller: "barclamp"
-  get "crowbar/:barclamp/1.0/status", action: "status", controller: "barclamp"
-  delete "crowbar/:barclamp/1.0/:id", action: "delete", controller: "barclamp"
-  get "crowbar/:barclamp/1.0/:id", action: "show", controller: "barclamp"
-  get "crowbar/:barclamp", action: "versions", controller: "barclamp"
-  post "crowbar/:barclamp/1.0/:action/:id", controller: "barclamp"
+  delete "crowbar/:barclamp/1.0/proposals/:id(.:format)", action: "proposal_delete", controller: "barclamp"
```

Comment:
Metrics/LineLength: Line is too long. [107/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)